forge-gui/res/cardsfolder/upcoming/gornog_the_red_reaper.txt

```diff
@@ -0,0 +1,12 @@
+Name:Gornog, the Red Reaper
+ManaCost:2 R
+Types:Legendary Creature Minotaur Warrior
+PT:2/3
+K:Haste
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Warrior | ValidBlocker$ Creature.Coward | Description$ Cowards can't block Warriors.
+T:Mode$ AttackersDeclaredOneTarget | AttackedTarget$ Player | ValidAttackers$ Creature.Warrior+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigAnimate | TriggerDescription$ Whenever one or more Warriors you control attack a player, target creature that player controls becomes a Coward.
+SVar:TrigAnimate:DB$ Animate | ValidTgts$ Creature | TargetsWithDefinedController$ TriggeredAttackedTarget | TgtPrompt$ Select target creature that player controls | Types$ Coward | RemoveCreatureTypes$ True

```

Comment:
duration missing

Reply:
Ah, I actually missed that it's a permanent effect :)